content/departments/cloud/technical-docs/v2.0/onprem_data_migration.md

```diff
@@ -0,0 +1,249 @@
+# On-prem data migration to Cloud v2
+
+This process describes the current state of how to do a full data migration of an on-prem instance to a [Cloud v2](./index.md) instance.
+On-prem-to-Cloud data migrations are currently owned by [Implementation Engineering](../../../technical-success/ie/index.md), but the process is documented in Cloud as it pertains to Cloud infrastructure.
+
+> WARNING: This process is still a work-in-progress. For more details, see [RFC 760](https://docs.google.com/document/d/1IAgXmv2TbtU_rWXtph-KFc3qbqswREIAxO1rGALuoMQ/edit#) and the [tracking issue](https://github.com/sourcegraph/customer/issues/1525), or cc @bobheadxi (Robert Lin).
+> This page was adapted only on a best-effort basis for Cloud v2 from [the legacy MI v1.1 page](../v1.1/index.md) - it has currently not yet been thoroughly tested for Cloud v2.
+
+## Requirements
+
+To qualify for a data migration, the customer must:
+
+- have a Sourcegraph instance on v3.20.0 or later
+  - note: where possible, strongly encourage the customer to upgrade to their on-prem instance to the latest version of Sourcegraph first.
+- use databases on Postgres 11 or later
+- _not_ have [on-disk database encryption](https://docs.sourcegraph.com/admin/config/encryption) enabled
+- have the [latest release](https://github.com/sourcegraph/src-cli/releases) of [`src`](https://github.com/sourcegraph/src-cli)
+- have direct database access
+- have a site-admin access token for their instance
+
+An operator must:
+
+- have the [latest _build_](https://github.com/sourcegraph/src-cli/blob/main/DEVELOPMENT.md#development) of [`src`](https://github.com/sourcegraph/src-cli) installed
+- have the `gcloud` CLI installed
+
+## Process
+
+### Prepare instance
+
+The customer should add a non-dismissible site notice to their instance in global settings:
+
+```json
+{
+  "notices": [
+    {
+      "dismissible": false,
+      "location": "top",
+      "message": "🚨 A Sourcegraph instance migration is underway - changes to configuration might not be persisted, and performance may be affected, until the migration is finalized."
+    }
+  ]
+}
+```
+
+### Create snapshot contents
+
+#### Databases
+
+The customer should first be asked to create `pg_dump` exports of their Sourcegraph databases. Template commands can be generated with `src snapshot databases` for various configurations:
```

Comment:
Shouldn't we quiesce the Sourcegraph services prior to taking the pg_dump snapshots? At least in an ideal scenario where we have some flexibility in taking the instance down temporarily.

Reply:
Added a section to the doc here that mentions this!